server/aws-lsp-codewhisperer/package.json

```diff
@@ -27,7 +27,9 @@
         "test:unit": "ts-mocha -b 'src/**/*.test.ts'",
         "test:prettier": "prettier . --check",
         "test": "npm run lint && npm run test:prettier && npm run test:unit",
-        "fix:prettier": "prettier . --write"
+        "fix:prettier": "prettier . --write",
+        "prepack": "ts-node ../../script/link_bundled_dependencies.ts",
+        "postinstall": "node ./script/install_bundled_dep.js"
```

Comment:
I tested locally that this script is not necessary - npm is able to resolve packed bundled dependencies with the package.